src/components/utils/LayerManager.ts

```diff
@@ -89,6 +89,10 @@ class LayerManager {
     };
 
     private handleDocumentClick = (event: MouseEvent) => {
+        if (this.isToastClick(event)) {
```

Comment:
What problem you trying to solve?

Reply:
https://github.com/gravity-ui/uikit/issues/544

It is impossible to close Toast component avoiding Modal closing

Reply:
May be we should handle click on overlay (`<Modal/>`) instead of `document`?

Reply:
What do you mean ?

Reply:
I thought we can handle click without targeting specific elements, but after reviewing code i guess this will break Popup close logic